lib/src/base-styles/_nx-variables.scss

```diff
@@ -62,9 +62,9 @@
   --nx-border-grid: var(--nx-border-2x);
 
   // for dropdown
-  --nx-box-shadow-dropdown: 0 2px 12px 0 rgba(74, 77, 113, 0.2),
-                            0 1px 2px 0 rgba(91, 95, 141, 0.2),
-                            0 1px 1px 0 rgba(124, 128, 179, 0.2);
+  --nx-box-shadow-dropdown: 0 2px 12px 0 Hsla(232, 43%, 80%, 0.2),
+                            0 1px 2px 0 Hsla(232, 42%, 70%, 0.2),
+                            0 1px 1px 0 Hsla(231, 43%, 60%, 0.2);
```

Comment:
All of these are 231 in figma.  Moreover I'm assuming they're all supposed to be indigo and any variation from our standard indigo hue and saturation is a rounding error.  There are similar issues in the dark mode value for this variable.

Reply:
Confirmed with Design that it should be the Indigo. 
Updated here fa72e1ab71dd4515dd3824abac032458d8dbdfec.